langlib/langlib-test/src/test/resources/test-src/floatlib_test.bal

```diff
@@ -415,6 +415,176 @@ function testFromStringNegative() {
     }
 }
 
+function testRound() {
+    assertEquality(float:round(123.123f, 2), 123.12f);
```

Comment:
The float suffix isn't required in these tests?

Reply:
changed